doc/nrf/gs_testing.rst

```diff
@@ -120,3 +120,93 @@ To connect to the nRF9160-based kit with LTE Link Monitor, perform the following
    .. note::
 
       In the case of nRF9160 DK, the reset button must be pressed to restart the device and to start the application.
+
+.. _gs_debugging:
+
+Debugging an application
+************************
+
+To debug an application, set up the debug session as described in `Debugging an application`_ in the |nRFVSC| documentation.
+
+If you use a multi-core SoC, for example from the nRF53 Series, you need a single debug session to debug the application core firmware.
+To debug the firmware running on the network core, you need to set up two separate debug sessions: one for the network core and one for the application core.
```

Comment:
First it is mentioned that a single debug session is needed for debugging the app core is described, but right after it is mentioned that two separate debug sessions are needed for debugging both cores.

I guess that actually means that the first line is only valid for only app core debugging.
So how about:
```suggestion
If you use a multi-core SoC, for example from the nRF53 Series, and you only wish to debug the application core firmware then a single debug session is sufficient.
To debug the firmware running on the network core, you need to set up two separate debug sessions: one for the network core and one for the application core.
```